src/cache_manager.cc

```diff
@@ -244,20 +245,20 @@ CacheManager::ParseHeaders(const HttpRequest * request, Mgr::ActionParams &param
     // TODO: use the authentication system decode to retrieve these details properly.
 
     /* base 64 _decoded_ user:passwd pair */
-    const char *basic_cookie = request->header.getAuth(Http::HdrType::AUTHORIZATION, "Basic");
+    const SBuf basic_cookie(request->header.getAuthToken(Http::HdrType::AUTHORIZATION, "Basic"));
```

Comment:
If possible, please use `auto` here and in a similar place elsewhere in the PR.

```suggestion
    const auto basic_cookie(request->header.getAuthToken(Http::HdrType::AUTHORIZATION, "Basic"));
```

Reply:
Done